pkg/quota/v1/install/update_filter.go

```diff
@@ -44,3 +51,17 @@ func DefaultUpdateFilter() func(resource schema.GroupVersionResource, oldObj, ne
 		return false
 	}
 }
+
+// hasResourcesChanged function to compare resources in container statuses
+func hasResourcesChanged(oldPod *v1.Pod, newPod *v1.Pod) bool {
+	if len(oldPod.Status.ContainerStatuses) != len(newPod.Status.ContainerStatuses) {
+		return false
+	}
+	for i, oldStatus := range oldPod.Status.ContainerStatuses {
```

Comment:
Can you check `InitContainerStatuses` too? Sidecar support is in-flight, but if your PR merges first, it will just be a no-op.
https://github.com/kubernetes/kubernetes/pull/128367

Reply:
fixed